kafka/config.go

```diff
@@ -183,6 +183,14 @@ func (m ConfigMap) convert() (cConf *C.rd_kafka_conf_t, err error) {
 // If the key is not found defval is returned.
 // If the key is found but the type is mismatched an error is returned.
 func (m ConfigMap) get(key string, defval ConfigValue) (ConfigValue, error) {
+	if strings.HasPrefix(key, "{topic}.") {
```

Comment:
librdkafka v0.11.0 added support for automatic fall-thru of global -> default.topic.config properties: any property set on the global object that turns out to be unknown will be attempted on the default topic config.
That makes this syntax obsolete, but since the Set() API already supports it we can't remove it, and it was better to add support to the corresponding Get() call as well.
I don't foresee anything but the simplest CLI utils to use this feature though.